doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -3,191 +3,54 @@
 
 This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
 
-ifdef::DM[]
+== Ability to only enable decision management components and functionality
 
-== {PRODUCT} {ENTERPRISE_VERSION} Kogito decision services support
+You can now configure the deployment to only enable decision management related components and functionality so that jBPM and case management features are disabled.
 
-=== IBM Power support in {OPENSHIFT} 4.10
+== Ability to use a custom registry and specific images
 
-Starting with {PRODUCT} {ENTERPRISE_VERSION}, IBM Power supports {OPENSHIFT} 4.10 using Open JDK 11. For a complete list of supported configurations, see https://access.redhat.com/articles/3354301[Red Hat Decision Manager 7 Supported Configurations].
+You can now configure the custom registry to use specific images during the configuration of the environment.
 
-endif::DM[]
+== Ability to output error logs
 
-== Ability to send cookies in the REST work item handler
+You can now set the `org.kie.jbpm.event.emitters.eagerInit` property to `true` to output the error log when {KIE_SERVER] starts and initializes the Kafka emitter.
 
-You can now send cookies to the REST work item handler using the `Cookie` header and cookie parameters in a REST API as shown in the following example:
+== Support for Quarkiverse File Vault and keystore files
```

Comment:
Shouldn't the title mention this is in the context of PIM?

Reply:
This should also not be in the RHDM release notes

Reply:
Mentioned PIM in the title